<?xml version="1.0"?>
<launch>

    <arg name="camera_info_directory" value="$(find tis_camera_driver)/intrinsics/external.yaml" />  <!-- Intrinsic Calibration file directory -->

    <node pkg="tis_camera_driver" type="tis_camera" name="tis_camera_node" output="screen">
        <param name="camera_info_url" type="string" value="$(arg camera_info_directory)" />
        <rosparam command="load" file="$(find tis_camera_driver)/config/configuration.yaml" />
    </node>

</launch>
